js/bybit.js

```diff
@@ -1653,7 +1653,10 @@ module.exports = class bybit extends Exchange {
         const timestamp = this.parse8601 (this.safeString (order, 'created_at'));
         const id = this.safeString2 (order, 'order_id', 'stop_order_id');
         const type = this.safeStringLower (order, 'order_type');
-        const price = this.safeString (order, 'price');
+        let price = undefined;
```

Comment:
@kroitor Although we're setting it as undefined, the price will have a value because it's calculated inside safeorder using the amount and cost